.github/workflows/sonar.yml

```diff
@@ -2,12 +2,10 @@ name: Sonar
 'on':
   push:
     branches:
-      - main
-  pull_request_target:
-    types:
-      - opened
-      - synchronize
-      - reopened
+      - master
```

Comment:
This should be `main` now that we renamed `master` into `main` (or the workflows won't run)